src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,115 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final double LOAD_FACTOR = 0.75;
+    private static final int GROWTH_RATE = 2;
+    private Node<K, V>[] arrayNodes;
+    private int size;
+    private int actualCapacity;
+    private int threshold;
+
+    private static class Node<K, V> {
+        private int hashKey;
+        private K key;
+        private V value;
+        private Node<K, V> next;
+
+        public Node(int hashKey, K key, V value) {
+            this.hashKey = hashKey;
+            this.key = key;
+            this.value = value;
+        }
+    }
```

Comment:
It's will be better, if you move it to the end of class)
And modifier "static"  is optional

Reply:
Okay I'll move it